tests/concurrent_test.go

```diff
@@ -47,16 +47,16 @@ func TestProxy_ConcurrencyGRPC(t *testing.T) {
 	clientset := runAgent(proxy.agent, stopCh)
 	waitForConnectedServerCount(t, 1, clientset)
 
-	// run test client
-	tunnel, err := client.CreateSingleUseGrpcTunnel(ctx, proxy.front, grpc.WithInsecure())
-	if err != nil {
-		t.Fatal(err)
-	}
-
 	var wg sync.WaitGroup
 	verify := func() {
 		defer wg.Done()
 
+		// run test client
```

Comment:
Note to reviewers: This test was reusing the tunnel across multiple requests, which we explicitly say not to do. It worked before because the test only sets up a single backend, and the FE request fits in a single data packet. This PR makes the failure more explicit when the connection is reused, which broke this test.